src/main/resources/services/webcruiterAdvertismentList/webcruiterAdvertismentList.ts

```diff
@@ -0,0 +1,22 @@
+import { request } from '/lib/http-client'
+import { type XmlParser } from '/lib/types/xmlParser'
+
+const xmlParser: XmlParser = __.newBean('no.ssb.xp.xmlparser.XmlParser')
+
+export function fetchWebcruiterAdvertismentListRSSFeed(url: string) {
```

Comment:
Ordet "Advertisement" er konsekvent feilstavet i PR'en - tror du kan gjøre en search and replace så er det kjapt fikset :)